test/support/docker-compose-test.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  jellyfish:
+    image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-edge}"
```

Comment:
As above, I am not convinced that edge should be the default

Reply:
I think we may change it later, when an appropriate tag(s) is created for jellyfish.

For now we have to work with the latest version, which is `edge`.